evaluate/report/csv.go

```diff
@@ -121,3 +181,164 @@ func WriteEvaluationRecord(resultPath string, model model.Model, language langua
 
 	return nil
 }
+
+// loadEvaluationRecords reads and returns the evaluation records from the evaluation CSV file.
+func loadEvaluationRecords(resultPath string) (evaluationRecords EvaluationRecords, err error) {
+	evaluationFilePath := filepath.Join(resultPath, "evaluation.csv")
```

Comment:
would not hand the result path but only the file where the raw records are. less hardcoded values